core/server/api_container/server/startosis_engine/kurtosis_instruction/tasks/run_python.go

```diff
@@ -321,6 +321,10 @@ func (builtin *RunPythonCapabilities) FillPersistableAttributes(builder *enclave
 	builder.SetType(RunPythonBuiltinName)
 }
 
+func (builtin *RunPythonCapabilities) Description() string {
+	return "Running a one time python script"
```

Comment:
Can we make this `Running Python script`? (also just a reminder that for consistency we use proper-casing for proper nouns like Python, Docker, Kubernetes, etc.)